Looks like this broke mac-only code:
```
error[E0433]: failed to resolve: use of undeclared type `KeyState`
   --> ports/winit/headed_window.rs:186:27
    |
186 |         if event.state == KeyState::Down {
    |                           ^^^^^^^^ use of undeclared type `KeyState`

error[E0433]: failed to resolve: use of undeclared type `KeyState`
   --> ports/winit/headed_window.rs:209:27
    |
209 |         if event.state == KeyState::Down && event.key == Key::Unidentified {
    |                           ^^^^^^^^ use of undeclared type `KeyState`

error[E0433]: failed to resolve: use of undeclared type `Key`
   --> ports/winit/headed_window.rs:209:58
    |
209 |         if event.state == KeyState::Down && event.key == Key::Unidentified {
    |                                                          ^^^ use of undeclared type `Key`

error[E0433]: failed to resolve: use of undeclared type `KeyState`
   --> ports/winit/headed_window.rs:214:34
    |
214 |         } else if event.state == KeyState::Up && event.key == Key::Unidentified {
    |                                  ^^^^^^^^ use of undeclared type `KeyState`

error[E0433]: failed to resolve: use of undeclared type `Key`
   --> ports/winit/headed_window.rs:214:63
    |
214 |         } else if event.state == KeyState::Up && event.key == Key::Unidentified {
    |                                                               ^^^ use of undeclared type `Key`

error[E0433]: failed to resolve: use of undeclared type `Key`
   --> ports/winit/headed_window.rs:223:25
    |
223 |         if event.key != Key::Unidentified {
    |                         ^^^ use of undeclared type `Key`

error[E0433]: failed to resolve: use of undeclared type `KeyState`
   --> ports/winit/headed_window.rs:699:27
    |
699 |         if input.state != KeyState::Down {
    |                           ^^^^^^^^ use of undeclared type `KeyState`

error[E0433]: failed to resolve: use of undeclared type `Key`
   --> ports/winit/headed_window.rs:707:13
    |
707 |             Key::Character(ref k) => match &**k {
    |             ^^^ use of undeclared type `Key`

error[E0412]: cannot find type `KeyboardEvent` in this scope
   --> ports/winit/headed_window.rs:61:32
    |
61  |     last_pressed: Cell<Option<(KeyboardEvent, Option<VirtualKeyCode>)>>,
    |                                ^^^^^^^^^^^^^
    |
   ::: /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.28.3/src/event.rs:844:1
    |
844 | pub struct KeyboardInput {
    | ------------------------ similarly named struct `KeyboardInput` defined here
    |
help: a struct with a similar name exists
    |
61  |     last_pressed: Cell<Option<(KeyboardInput, Option<VirtualKeyCode>)>>,
    |                                ~~~~~~~~~~~~~
help: consider importing this struct
    |
7   | use keyboard_types::KeyboardEvent;
    |

error[E0412]: cannot find type `Key` in this scope
  --> ports/winit/headed_window.rs:64:48
   |
64 |     keys_down: RefCell<HashMap<VirtualKeyCode, Key>>,
   |                                                ^^^ not found in this scope
   |
help: consider importing this enum
   |
7  | use keyboard_types::Key;
   |

error[E0433]: failed to resolve: use of undeclared type `KeyboardEvent`
   --> ports/winit/headed_window.rs:182:14
    |
182 |             (KeyboardEvent::default(), None)
    |              ^^^^^^^^^^^^^ use of undeclared type `KeyboardEvent`
    |
help: a struct with a similar name exists
    |
182 |             (KeyboardInput::default(), None)
    |              ~~~~~~~~~~~~~
help: consider importing this struct
    |
7   | use keyboard_types::KeyboardEvent;
    |

error[E0433]: failed to resolve: use of undeclared type `Key`
   --> ports/winit/headed_window.rs:184:21
    |
184 |         event.key = Key::Character(ch.to_string());
    |                     ^^^ use of undeclared type `Key`
    |
help: consider importing this enum
    |
7   | use keyboard_types::Key;
    |

error[E0412]: cannot find type `KeyboardEvent` in this scope
   --> ports/winit/headed_window.rs:698:45
    |
698 |     fn handle_xr_translation(&self, input: &KeyboardEvent) {
    |                                             ^^^^^^^^^^^^^
    |
   ::: /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/winit-0.28.3/src/event.rs:844:1
    |
844 | pub struct KeyboardInput {
    | ------------------------ similarly named struct `KeyboardInput` defined here
    |
help: a struct with a similar name exists
    |
698 |     fn handle_xr_translation(&self, input: &KeyboardInput) {
    |                                             ~~~~~~~~~~~~~
help: consider importing this struct
    |
7   | use keyboard_types::KeyboardEvent;
    |
```

--- a/ports/winit/headed_window.rs
+++ b/ports/winit/headed_window.rs
@@ -16,7 +16,7 @@ use winit::window::Icon;
 use winit::event::{ElementState, KeyboardInput, MouseButton, MouseScrollDelta, TouchPhase, VirtualKeyCode};
 #[cfg(any(target_os = "linux", target_os = "windows"))]
 
-use keyboard_types::{Key, KeyState, KeyboardEvent};
+use servo::keyboard_types::{Key, KeyState, KeyboardEvent};
 use servo::compositing::windowing::{AnimationState, MouseWindowEvent, WindowEvent};
 use servo::compositing::windowing::{EmbedderCoordinates, WindowMethods};
 use servo::embedder_traits::Cursor;

--- a/ports/winit/headed_window.rs
+++ b/ports/winit/headed_window.rs
@@ -14,7 +14,6 @@ use euclid::{
 #[cfg(any(target_os = "linux", target_os = "windows"))]
 use winit::window::Icon;
 use winit::event::{ElementState, KeyboardInput, MouseButton, MouseScrollDelta, TouchPhase, VirtualKeyCode};
-#[cfg(any(target_os = "linux", target_os = "windows"))]
 
 use servo::keyboard_types::{Key, KeyState, KeyboardEvent};
 use servo::compositing::windowing::{AnimationState, MouseWindowEvent, WindowEvent};

